src/connector.cpp

```diff
@@ -5139,11 +4984,7 @@ void HostConnector::hostSwitchPreset(const Current& prev)
                 {
                     const HostBlockPair hbp = _mapper.remove(prev.preset, row, bl);
 
-                    if (hbp.id != kMaxHostInstances)
-                        _host.remove(hbp.id);
-
-                    if (hbp.pair != kMaxHostInstances)
-                        _host.remove(hbp.pair);
+                    hostRemoveBlockPair(hbp);
```

Comment:
the check for != max is wanted here, because I think its useful to have an assertion inside hostRemoveBlockPair so we catch bad usage.
on this case it is the exception where it can be invalid, due to there being no plugin loaded